desktop/core/src/desktop/lib/botserver/api.py

```diff
@@ -57,6 +61,27 @@ def send_message(request):
     'ok': slack_response.get('ok'),
   })
 
+@login_notrequired
+@api_error_handler
+def generate_slack_install_link(request):
+  response = {'status': 0}
+  install_link = 'https://api.slack.com/apps?new_app=1&manifest_yaml='
+
+  hostname = request.POST.get('hostname')
+
+  with open(os.path.join(BASE_DIR, 'tools/slack/manifest.yml')) as manifest:
+    data = yaml.safe_load(manifest)
+
+    data['features']['unfurl_domains'] = [hostname]
+    data['settings']['event_subscriptions']['request_url'] = 'https://' + hostname + '/desktop/slack/events/'
+
+    changed_data = yaml.safe_dump(data)
+    install_link += quote_plus(changed_data)
+
+    response['link'] = install_link
+
+  return JsonResponse(response)
```

Comment:
Would it be simpler to just redirect to the url?

Reply:
@romainr  trying with `return redirect(install_link)` but when testing `response.status_code` , its failing with 403 status code. Not sure where am I missing for a correct redirect to the install link. Any hints?